salt/modules/selinux.py

```diff
@@ -616,7 +616,7 @@ def _fcontext_add_or_delete_policy(
     if "add" == action:
         # need to use --modify if context for name file exists, otherwise ValueError
         filespec = re.escape(name)
-        cmd = f"semanage fcontext -l | egrep '{filespec} '"
+        cmd = f"semanage fcontext -l | grep -E '{filespec}'"
```

Comment:
The trailing space after `{filespec}` is an important part of the regex and must be kept

Reply:
thanks for that, will adjust, test needs that space then :+1: